p256-crypto/src/multi_signature.rs

```diff
@@ -13,6 +15,8 @@ use crate::application_crypto::p256::{Public, Signature};
 
 #[cfg_attr(feature = "std", derive(Serialize, Deserialize))]
 #[derive(Eq, PartialEq, Clone, Encode, Decode, MaxEncodedLen, RuntimeDebug, TypeInfo)]
+// SBP-M1 review: `MultiSignature` is a misleading name -> similar to multisig that is responsible for handling e.g. n of m signatures
+// I would use different name for it
 pub enum MultiSignature {
     /// An Ed25519 signature.
     Ed25519(ed25519::Signature),
```

Comment:
This types is called `MultiSignature` because it mimics the type of the same name in substrate: 

https://github.com/paritytech/substrate/blob/1837f423b494254e1d27834b1c9da34b2c0c2375/primitives/runtime/src/lib.rs#L240

It functions the same way as the above liked type but additionally it adds the `P256` signature case